<launch>
    <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
        <remap from="imu_data" to="/mobile_base/sensors/imu_data"/>
        <param name="output_frame" value="odom_combined"/>
        <param name="base_footprint_frame" value="base_footprint"/>
        <param name="freq" value="100000.0"/>
        <param name="sensor_timeout" value="1.0"/>  
        <param name="odom_used" value="true"/>
        <param name="imu_used" value="true"/>
        <param name="vo_used" value="false"/>
    </node>

    <node pkg="bot" type="pos_info" name="pos_info" ns="auto_ctrl"/>
    <node pkg="bot" type="depth_info" name="depth_info" ns="auto_ctrl"/>
    <node pkg="bot" type="scan_info" name="scan_info" ns="auto_ctrl"/>
    <node pkg="bot" type="bot_control" name="bot_control" ns="auto_ctrl" output="screen"/>
</launch>